data/mods/Aftershock/items/crafting_scrap/biomaterial_scrap.json

```diff
@@ -20,23 +20,23 @@
     "id": "afs_biomaterial_3",
     "copy-from": "afs_scrap_1",
     "name": { "str_sp": "artificial muscle fibers" },
-    "description": "Lab grown or bioprinted muscle fibers, much denser and with higher performance ratios than baseline human muscles.",
+    "description": "Lab-grown or bioprinted muscle fibers; much denser and with higher performance ratios than baseline human muscles.",
     "looks_like": "scrap"
   },
   {
     "type": "GENERIC",
     "id": "afs_biomaterial_4",
     "copy-from": "afs_scrap_4",
     "name": { "str_sp": "self healing polymers" },
-    "description": "Materials capable of taking environmental chemicals and utilizing them for self repair, be careful you don't make grey goo.",
+    "description": "Materials capable of taking environmental chemicals and utilizing them for self-repair; be careful you don't make grey goo.",
     "looks_like": "scrap"
   },
   {
     "type": "GENERIC",
     "id": "afs_biomaterial_5",
```

Comment:
I suggest swapping the names and descriptions for afs_biomaterial_4 and afs_biomaterial_5 - or even moving afs_biomaterial_5 to 3, 3 to 4, and 4 to 5. In terms of levels of difficulty to create, this would be more appropriate. The autologous totipotent tissue culture actually works well for the smart purifier.

Reply:
I'm going to punt this question to @Mom-Bun. Thoughts?

Reply:
Hi, personally I feel like the description can be looked at later, but right now I'd prefer this PR just touch the recipe for now.

Reply:
Understand; no problem.